### ISSUE TITLE
Server crashes with NullPointer

### ISSUE THREAD

**Issue by IcyPalm** [IcyPalm/Hanze-TwoPlayerGameServer#33]
The server is not fully stable and sometimes crashes with this error message:
```
Exception in thread "ClientInputHandler-Thread" java.lang.NullPointerException
        at nl.hanze.gameserver.server.ClientManager.removeClient(ClientManager.java:63)
        at nl.hanze.gameserver.server.ClientInputHandler.run(ClientInputHandler.java:63)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by goto-bus-stop** [comment]
Looks a lot like a double disconnect in GameServer.java around line 200

**Comment by roffel** [comment]
Why that double Error catch without doing anything anyways.

```java
		} catch (IOException e) {
			;
		} catch (Exception e) {
			;
		}
```

Could be:

```java
		} catch (Exception e) {
			System.out.println(e);
		}
```

I ❤️  Hanze code.

**Comment by IcyPalm** [comment]
Fixed with #36
